src/main/java/org/fcrepo/auth/webac/WebACAuthorizationDelegate.java

```diff
@@ -38,6 +38,8 @@
 import org.fcrepo.auth.roles.common.AbstractRolesAuthorizationDelegate;
 import org.fcrepo.auth.roles.common.AccessRolesProvider;
 import org.fcrepo.kernel.api.services.NodeService;
+
+import org.modeshape.jcr.ModeShapePermissions;
```

Comment:
Also, do we need a FedoraPermissions interface that inherits the ModeshapePermissions, instead of directly using it?